### DISCUSSION TITLE
Api domain get challenge info

### DISCUSSION THREAD

**Discussion by advany** [vercel/community#764] [Help] [answered]
So when I add the domains to a project there is no info on the challenge.

Where can I get if an domain needs to point to a certain ip or certain dns server (in case of wildcard domains) or txt record (if owned by a different team)?

I couldnt find it anywhere

**Comment by amyegan** [comment, marked as answer]
You can follow our [support article about adding a custom domain to a Vercel project](https://vercel.com/support/articles/how-do-i-add-a-custom-domain-to-my-vercel-project) if you have not already added the domain. You'll need to use Vercel nameservers if you want to [use wildcard domains with Vercel](https://vercel.com/blog/wildcard-domains). The Domain Settings page will tell you what nameservers or A records to add to your domain provider when using an external domain.

![Screen Shot 2022-08-03 at 11 53 47 AM](https://user-images.githubusercontent.com/8247105/182654404-0d149ed2-6819-4a94-8b5a-47eb08e65932.png)

![image](https://user-images.githubusercontent.com/8247105/182655561-71ad78bd-5e36-4b64-adef-34322f52a388.png)
